configs/Lightning-AI/pl_torch-develop.yaml

```diff
@@ -0,0 +1,33 @@
+target_repository:
+  HTTPS: https://github.com/Lightning-AI/lightning.git
+  # checkout a particular branch or a tag
+  checkout: master
+  # define installing package extras
+  install_extras: dev
+  # copy some  test from the target repository
+  copy_tests:
+    - tests/tests_pytorch
+
+contact:
+  slack:
+    # Carlos Mocholí
+    - U019R2ZS25T
+
+env:
+  # let's limit the installation only to Pytorch-Lightning
+  PACKAGE_NAME: pytorch
+
+dependencies:
+  - name: torch
+    install_flags:
+      - "-f https://download.pytorch.org/whl/nightly/torch_nightly.html"
+      - "--pre"
+
+testing:
+  # additional pytest arguments
+  pytest_args: --strict
+
+runtimes:
+  - {os: "ubuntu-22.04", python: "3.10"}
+  - {os: "macOS-12", python: "3.10"}
+#  - {os: "windows-2022", python: "3.10"}
```

Comment:
why no windows?

Reply:
again just copied and let's keep this future integration simple, but if you want I am fine to enable it right now :)

Reply:
I'd prefer to have a full matrix for everything :) Just so we make sure we don't miss anything while we update it. It shouldn't be too much as we only have one python and one torch version per OS

Reply:
```suggestion
  - {os: "windows-2022", python: "3.10"}
```